qr_generator/lib/utils/api_client.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:dio/dio.dart';
+
+class ApiClient {
+  late Dio dio;
+  ApiClient() {
+    setupDio();
+  }
+
+  setupDio() async {
+    // Change baseUrl  to localhost:3030 if you are using iOS Simulator,
+    // on android use the local ip
+    BaseOptions options = BaseOptions(
+      connectTimeout: 5000,
+      receiveTimeout: 5000,
+      baseUrl: 'http://192.168.86.45:3030/', //192.168.86.30
+    );
+    dio = Dio(options);
+  }
+
+//Dynamic get
+  Future<dynamic> getDio(String url, {Map<String, dynamic>? parameters}) async {
```

Comment:
**suggestion**: try to avoid returning `dynamic`. Handle casts internally to avoid propagating unsafe `dynamic` which has no compile-time safety and is very error prone

Reply:
Thank you for the feedback, I'll take it into account in the future.